Kickstarter-iOS/Features/PaymentMethods/Controller/PaymentMethodsViewController.swift

```diff
@@ -45,6 +50,7 @@ internal final class PaymentMethodsViewController: UIViewController, MessageBann
     ]
 
     self.dataSource.deletionHandler = { [weak self] creditCard in
+      self?.viewModel.inputs.shouldCancelPaymentSheetAppearance(state: true)
```

Comment:
I noticed that we're not passing `false` to `shouldCancelPaymentSheetAppearance(state: Bool)` other than in the test. Can we remove this param and assume that if whenever this input is called, we should cancel the payment sheet?

Reply:
We don't send in `false` via the view controller, you're right but we do send it in through the view model `PaymentMethodsViewModel`. In fact it's default state is `false`. Also those signal's we merged in the above comment, one of them does map `false` to the `shouldCancelPaymentSheetAppearance` which is convenient. 

So just stepping back, see this as a signal that acts as an `input` _outside_ the view model and and `input` _inside_ the view model`.

Reply:
Ah I see. Cool thanks